Makefile

```diff
@@ -1,58 +1,59 @@
 #* Variables
 SHELL ?= /usr/bin/env bash
-PYTHON ?= python
-PYTHONPATH := `pwd`
-POETRY ?= poetry
-
-#* Poetry
-.PHONY: poetry-install
-poetry-install:
-	curl -sSL https://install.python-poetry.org | $(PYTHON) -
-
-.PHONY: poetry-remove
-poetry-remove:
-	curl -sSL https://install.python-poetry.org | $(PYTHON) - --uninstall
-
-.PHONY: poetry-plugins
-poetry-plugins:
-	$(POETRY) self add poetry-plugin-up
-
-.PHONY: poetry-env
-poetry-env:
-	$(POETRY) config virtualenvs.in-project true
+ACTIVEPYTHON = $(shell which python)
+
+#* UV
+.PHONY: setup
+setup:
+	@if ! command -v uv &> /dev/null; then \
+		curl -LsSf https://astral.sh/uv/install.sh | sh; \
+	else \
+		echo "uv is already installed"; \
+	fi
+
+.PHONY: remove
+remove:
+	@read -p "Are you sure you want to remove uv and all its data? [y/N] " confirm && \
+	if [ "$$confirm" = "y" ]; then \
+		uv cache clean; \
+		rm -r "$$(uv python dir)"; \
+		rm -r "$$(uv tool dir)"; \
+		rm ~/.cargo/bin/uv ~/.cargo/bin/uvx; \
```

Comment:
Isn't there a risk this wipes the user's entire cache/binaries, rather than project-specific ones? I'm not a user of cargo, so I don't care that much, but just in case it is used as cross-project tool, that wouldn't be nice for them.

Reply:
yeah I'll remove. there's this [uninstall poetry command](https://github.com/stac-extensions/mlm/blob/main/Makefile#L13), not sure if we even want to offer a remove uv command here since folks may install here and use across projects.